src/pages/Home.jsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+import Card from '../components/Card';
+import AddBook from '../components/AddBook';
+
+function Home() {
+  const [books, setBook] = useState([
+    {
+      id: 1,
+      category: 'Action',
+      name: 'The Hunger Games',
+      director: 'Suzanne Collins',
+      completed: '64%',
+      chapter: 'Chapter 17',
+    },
+    {
+      id: 2,
+      category: 'Science Fiction',
+      name: 'Dune',
+      director: 'Frank Herbert',
+      completed: '8%',
+      chapter: 'Chapter 3: A Lession',
+    },
+    {
+      id: 3,
+      category: 'Economy',
+      name: 'Capital in the Century',
+      director: 'Suzanne Collins',
+      completed: '0%',
+      chapter: 'Introduction',
+    },
+  ]);
+
+  const addBook = (newBook) => {
+    setBook([newBook, ...books]);
+  };
+
+  const handleDelete = (nid) => {
+    setBook(books.filter((todo) => todo.id !== nid));
+  };
+
+  return (
+    <>
+      {books.map((todo) => (
+        <Card
+          key={todo.id}
+          category={todo.category}
+          name={todo.name}
+          director={todo.director}
+          completed={todo.completed}
+          chapter={todo.chapter}
+          deleteBook={() => handleDelete(todo.id)}
+        />
+      ))}
+      <div className="border-b-2 border-b-gray-200 pt-5" />
+      <AddBook handleAdd={addBook} />
+    </>
+  );
+}
```

Comment:
Good job so far! 🤗

According to the project requirements. The structure of our component should look this way:

Screenshot
![image](https://github.com/evansnyamekye/bookstore/assets/78251927/9b40f6d1-27e0-44a2-909c-f0610cec7ec3)
========
- The `<Books>` component should contain the following components:

  - `<BookList>`
    - Renders a list of books using the `<Book>` component
  - `<BookForm>`
    - Renders a form with inputs for the title and the author of the book
  

- `<Book>` component is used in the `<BookList>` component
    - Takes the book data as a prop and renders the title and the author
    - Has a button for removing a book

example

```JavaScript
const Books = () => {
...

  const books = useSelector((state) => state.books);

  return (
    <div>
      <BookList books={books} />
      <AddBook />
    </div>
  );
};
```

Please implement that 👍